doc/build/changelog/unreleased_20/11870.rst

```diff
@@ -0,0 +1,9 @@
+.. change::
+    :tags: bug, mysql
+    :tickets: 11870
+
+    Fixed discepancy between mysqlclient and mariadbconnector where the
+    latter didn't merge url query options with the other args preventing options
```

Comment:
**Michael Bayer** (zzzeek) wrote:

I think more succinctly the issue was mariadbconnector ignored all arguments that were not ints or booleans

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5505

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5505